hazelcast/src/test/java/com/hazelcast/spi/impl/NodeEngineTest.java

```diff
@@ -60,7 +60,7 @@ public void getServiceOrNull_whenNullName() {
 
     @Test
     public void getServiceOrNull_whenNonExistingService() {
-        Object sharedService = nodeEngine.getServiceOrNull("notexist");
+        Object sharedService = nodeEngine.getServiceOrNull("nonexistent");
```

Comment:
```suggestion
        Object sharedService = nodeEngine.getServiceOrNull("nonexistent");
```
```suggestion
        Object sharedService = nodeEngine.getServiceOrNull("notexist");
```

Reply:
There appear to be two distinct suggestions here. I'm not quite certain what the first suggestion is showing (it feels like an error). I've tentatively batched the second which I presume is a revert.